robot_sdk/wrappers/ec_wrapper.py

```
@@ -67,43 +77,45 @@ def get_joint_angles(self) -> Optional[List[float]]:
-    def move_to_pose(self, pose: List[float]) -> Optional[bool]:
+    def move_to_pose(self, pose: List[float], tool_vel: float = 0.20, tool_acc: float = 0.01, user_num: int = 0) -> Optional[tuple]:
         '''
         Move the robot to a desired Cartesian pose.
         '''
         try:
             if not isinstance(pose, list) or len(pose) != 6:
                 print("Error: Pose must be a list of 6 values [x, y, z, rx, ry, rz]")
-                return False
-            return self.client.movel(points=pose)
+                return False, None
+            pose = [pose[0]*1000, pose[1]*1000, pose[2]*1000] + pose[3:]  # Convert m to mm
```

Ruff (0.12.2)

Consider iterable unpacking instead of concatenation

(RUF005)